packages/neuron-wallet/src/database/chain/migrations/1690361215400-ResetSyncProgressPrimaryKey.ts

```diff
@@ -1,15 +1,15 @@
 import {MigrationInterface, QueryRunner} from "typeorm";
```

Comment:
Migration file should not be changed

Reply:
Because this migration has import `SyncProgress`, the current migration can not find these columns if the DB migrates from the start.
```
  @Column()
  lightStartBlockNumber: number = 0

  @Column()
  localSavedBlockNumber: number = 0

  @Column()
  syncedBlockNumber: number = 0
```

Reply:
I didn't get the point, but the update in the migration won't be executed. Every performed migration will be recorded in the database, so even the script has been updated, it won't be run again.

Reply:
Of course, the migration will not execute twice. But If the user installs the Neuron for the first time or deletes the old DB file, the migration file will execute. But currently the class `SyncProgress` has been changed. 
`const syncProgresses = await queryRunner.manager.find(SyncProgress)` This statement will throw an error because the next migration that changes `SyncProgress`'s field name has not been executed yet.

Reply:
Got it